pytorch_lightning/trainer/training_loop.py

```diff
@@ -876,11 +877,22 @@ def build_train_args(self, batch, batch_idx, opt_idx, hiddens):
                 )
 
         # pass hiddens if using tbptt
-        if self.trainer.truncated_bptt_steps is not None:
+        if self._truncated_bptt_enabled():
```

Comment:
shall this also be a property?
```suggestion
        if self.truncated_bptt_enabled:
```